vm/src/stdlib/symtable.rs

```diff
@@ -129,12 +136,23 @@ impl PySymbolTable {
 
     #[pymethod(name = "get_symbols")]
     fn get_symbols(&self, vm: &VirtualMachine) -> PyResult {
-        let symbols = self
-            .symtable
-            .symbols
-            .values()
-            .map(|s| (PySymbol { symbol: s.clone() }).into_ref(vm).into_object())
-            .collect();
+        let mut symbols: Vec<PyObjectRef> = vec![];
```

Comment:
Is there any reason this can't still be `self.symtable.symbols.values().map(...).collect()`? That's generally more idiomatic than `let mut v: Vec` and a for loop pushing values to it.

Reply:
Done, Thank you.
I'm just worried about whether too many chain calls will affect readability.